### DISCUSSION TITLE
Header design with logged in user

### DISCUSSION THREAD

**Discussion by Andra-Masheh** [TDRA-ae/aegov-dls#72] [General]
Entities have been facing challenges on how should the header look after a user logs in into their platform, will there be a display icon for the user identity , where the username will appear and if its cooperate with long name or a person who have multiple accounts how it should appear

**Comment by moiz-conceptlz** [comment]
Hello @Andra-Masheh 

This is a great discussion, and it is also open to the community to contribute to.

Let me perhaps further set the scenarios;

1. A user has logged into a particular website using UAE Pass, and there is an entire application that has been designed with a UX relative to the application's purpose. 
2. User - who is currently logged in - goes back to the primary website and here, the user need to be indicated that they are already logge in .. 

Is this the scenario? 

or .. are we talking about "how should the header look on a particular application system after a user is logged in - which is not related to the primary website"?

**Reply by Andra-Masheh** [reply]
Hello @moiz-conceptlz  , 

the scenario is weather the user was logged in to a service and went back to the primary website, in this case the user should appear as a logged in user, the header should contain a section to show the user name and how he can access his dashboard or logout

**Reply by moiz-conceptlz** [reply]
In this case, I would say that the icon that shows the user icon - which leads to the login page - this should be changed to the Avatar component and perhaps with the indicator of logged in. From the https://designsystem.gov.ae/docs/components/avatar, I would suggest the "Avatar with Indicator". 

Now, you would also add the Dropdown component on the Avatar. Following https://designsystem.gov.ae/docs/components/dropdown, you would select "With header, divider and icon" and you may also style this dropdown with whatever is required in the dropdown.

If the user's name must be displayed, and there is enough space to display it, then the entity must follow https://designsystem.gov.ae/docs/patterns/name as a pattern on how to display the name. 

Is this helpful?

**Reply by Andra-Masheh** [reply]
@moiz-conceptlz thank you
